### PR TITLE
Run3-gex176D Modify the RPC numbering scheme to avoid wrong DetId assignment

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify the RPC numbering scheme to avoid wrong DetId assignment

#### PR validation:

Avoid crash in the DIgin step due to RPC wrong DetId

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Need to be back ported to 14_0_X

### CODE REVIEW

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2024FlatMinus10PercentXML_cfi.py [hunk 1]
```diff
@@ -252,7 +252,7 @@
         'Geometry/MuonCommonData/data/mfshield/2017/v2/mfshield.xml',
     )+
     cms.vstring(
-        'Geometry/MuonCommonData/data/muonNumbering/2021/v5/muonNumbering.xml',
```
Comment (civanch) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2024FlatMinus10PercentXML_cfi.py:255]: @bsunanda , are you sure if we need this file? For me would be OK if these type of files are changed only for Run-3 legacy.

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2024FlatMinus05PercentXML_cfi.py [hunk 2]
```diff
@@ -252,7 +252,7 @@
         'Geometry/MuonCommonData/data/mfshield/2017/v2/mfshield.xml',
     )+
     cms.vstring(
-        'Geometry/MuonCommonData/data/muonNumbering/2021/v5/muonNumbering.xml',
+        'Geometry/MuonCommonData/data/muonNumbering/2021/v6/muonNumbering.xml',
```
Comment (civanch) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2024FlatMinus05PercentXML_cfi.py:255]: @bsunanda , why not ...../muonNumbering/2024/v1/muonNumbering.xml ?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by srimanob** [comment]
type RPC

**Comment by srimanob** [comment]
Hi @civanch 

Do we have study from RPC to confirm this? I would prefer to have it attach to this PR. Thanks.

**Comment by srimanob** [comment]
+Upgrade

Propose to merge as request in https://github.com/cms-sw/cmssw/pull/44065#issuecomment-1963627578

However, I still think that the confirmation of RPC is needed to attach to the PR. I don't think it is healthy to merge PR with only wording on removing crash. PR description does not explain anything to fix issue and how to test. Note that, this PR can't be tested with short or long matrix as we don't have a workflow to handle XML geometry for Run-3. From the email discussion with RPC, I understand that we should see something from RPC.

**Comment by civanch** [comment]
@srimanob , it was agreed at https://indico.cern.ch/event/1387191/ that we do not produce DB pilot and backports to 14_0 until green light from RPC and GEM DPG. This PR should be merge to go faster in this direction.

**Comment by civanch** [comment]
type urgent

**Comment by civanch** [comment]
urgent

**Comment by srimanob** [comment]
By the way, should this pr be squashed first? So that we don't have history of 2021/v6 anymore.

**Comment by rappoccio** [comment]
+1

**Pull request merged**